ac/ac-train/src/Dashboards/MeanPerInterface.jsx

```diff
@@ -0,0 +1,126 @@
+// @flow
+
+import * as React from 'react';
+import Grid from 'material-ui/Grid';
+import Typography from 'material-ui/Typography';
+import Paper from 'material-ui/Paper';
+import { withStyles } from 'material-ui/styles';
+
+import {
+  VictoryChart,
+  VictoryBar,
+  VictoryTheme,
+  VictoryTooltip,
+  VictoryAxis
+} from 'victory';
+
+import { color, div } from './utils';
+
+const styles = theme => ({
+  root: theme.mixins.gutters({
+    paddingTop: 16,
+    paddingBottom: 16,
+    marginTop: theme.spacing.unit * 3
+  })
+});
+
+const MeanPerInterface = props => {
+  const { whichDash, data } = props;
+
+  const count = data['count'];
+  const dash = data[whichDash];
+
+  const interfaces = Object.keys(dash);
+
+  if (interfaces.length > 0) {
+    const allInterfaces = ['dragdrop', 'form', 'graphical', 'command'];
+
+    const coordinates = interfaces.map(int => {
+      if (whichDash === 'help') {
+        let countSum = 0;
+
```

Comment:
When asking for help before any data has been put in the dashboards (help on the first instance of the first interface) the dashboards crash with error: cannot read property 0 of undefined.